src/builder.cr

```diff
@@ -1,7 +1,13 @@
 module Mint
   class Builder
+    getter json
+
     def initialize
-      json = MintJson.parse_current
+      @json = MintJson.parse_current
+
+      terminal.measure "#{COG} Ensuring dependencies... " do
+        json.check_dependencies!(sources)
```

Comment:
Passing the sources are not needed since the `json` knows about it's directory so it's calculable from there, or might not be needed at all (see last comment).